irr/src/CIrrDeviceSDL.cpp

```diff
@@ -1016,25 +1036,26 @@ bool CIrrDeviceSDL::activateJoysticks(core::array<SJoystickInfo> &joystickInfo)
 	return false;
 }
 
-//! Get the display density in dots per inch.
-float CIrrDeviceSDL::getDisplayDensity() const
+void CIrrDeviceSDL::updateSizeAndScale()
 {
-	if (!Window)
-		return 0.0f;
-
-	int window_w;
-	int window_h;
+	int window_w, window_h;
 	SDL_GetWindowSize(Window, &window_w, &window_h);
 
-	int drawable_w;
-	int drawable_h;
+	int drawable_w, drawable_h;
 	SDL_GL_GetDrawableSize(Window, &drawable_w, &drawable_h);
```

Comment:
fwiw `SDL_GetWindowSizeInPixels` seems to be the replacement for this: https://github.com/libsdl-org/SDL/issues/7172

Reply:
yeah, but `SDL_GetWindowSizeInPixels` was only added in 2.26, so we'd need to fall back to `SDL_GL_GetDrawableSize` anyway